docs/project/constructor.md

```diff
@@ -0,0 +1,18 @@
+
+Constructor
+===========
+  
+{% swagger method = "c0nstruct0r" path = " " baseUrl = " " summary = "constructor" %}  
+{% swagger-description %}  
+Initialize the contract  
```

Comment:
Maybe giving more information about what is initialized in the constructor or what this Smart Contract, throw this Constructor, allow to do. 

Because a comment "Initialize the contract" is not giving any real information. In addition to that, as the Constructor is the first "Object" when you navigate in the Gitbook, It should contain a quick description of what do or what is this Smart Contract.

For this one, for example, using the description from the header / mainpage "Carbonable Project" to describe what is a project in Carbonable ecosystem/context and that is a collection of NFTs. Because some dev will go to ready only method. 

The same for the other constructor description. 

What do you think ?

Reply:
I disagree for 2 two reasons:
- the smart-contract doc is out of the scope of the only constructor function, this is the purpose of the smart-contract page (markdown only)
- in addition, the smart-contract doc contains "business" information, which must be kept outside of the doc imho

I agree to add few more information about initialization in term of code